controllers/userController.js

```diff
@@ -137,24 +128,47 @@ module.exports = {
   },
   addAddress: async (req, res) => {
     try {
-      const { address, city, state, pinCode, phoneNum, type } = req.body;
-      const data = req.tokenData;
-
-      await Address.create({
-        user_id: data.user_id,
+      const { address, city, state, pin_code, phone_no, userId } = req.body;
+      const addAddress = await addresses.create({
         address,
         city,
         state,
-        pinCode,
-        phoneNum,
-        type,
+        pin_code,
+        phone_no,
+        userId,
```

Comment:
are you getting this userId from the token or directly from the payload? if from payload, change it @akaashinno